docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -573,6 +573,18 @@ public class GreetingResourceTest {
 
 If you are using the `quarkus-hibernate-orm-panache` or `quarkus-mongodb-panache` extensions, check out the link:hibernate-orm-panache#mocking[Hibernate ORM with Panache Mocking] and link:mongodb-panache#mocking[MongoDB with Panache Mocking] documentation for the easiest way to mock your data access.
 
+== Starting services before the Quarkus application starts
+
+A very common need is to start some services on which you Quarkus application depends on, before the Quarkus application starts for testing. To address this need Quarkus provides `@io.quarkus.test.common.QuarkusTestResource` and `io.quarkus.test.common.QuarkusTestResourceLifecycleManager`.
+
+By simply annotating any test in the test-suite with `@QuarkusTestResource`, Quarkus will run the corresponding `QuarkusTestResourceLifecycleManager` before any tests are run.
```

Comment:
```suggestion
By simply annotating any test in the test suite with `@QuarkusTestResource`, Quarkus will run the corresponding `QuarkusTestResourceLifecycleManager` before any tests are run.
```

Reply:
Was on the fence about this, but I absolutely trust you on it :)